src/TruncateExpandable.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components'
+import Truncate from './Truncate'
+
+const TruncateExpandable = styled(Truncate)`
+  &:hover {
+    max-width: 10000px !important;
```

Comment:
I felt weird about this `!important` but it's in the [truncate styles](https://github.com/primer/css/blob/master/src/truncate/truncate.scss), so I wanted to mirror that.